ivm/src/allocator.rs

```diff
@@ -36,7 +36,7 @@ impl<'ivm> Allocator<'ivm> {
   }
 }
 
-impl<'ivm> IVM<'ivm> {
+impl<'ext, 'ivm> IVM<'ext, 'ivm> {
```

Comment:
minor nit, but I'd prefer if it was ordered as `'ivm, 'ext` (a global find & replace should suffice)